torchao/quantization/quantize_/workflows/int8/int8_tensor.py

```diff
@@ -292,6 +302,8 @@ def _(func, types, args, kwargs):
             block_size,
             self.dtype,
             act_quant_kwargs=self.act_quant_kwargs,
+            act_scale=self.act_scale,
```

Comment:
I guess slice doesn't work for static quant int8 before, can you add a test for that?